app/src/main/java/br/com/mgalhardo/guidebook/dagger/UiComponent.java

```diff
@@ -0,0 +1,12 @@
+package br.com.mgalhardo.guidebook.dagger;
+
+import br.com.mgalhardo.guidebook.dagger.module.presentation.PresenterModule;
+import br.com.mgalhardo.guidebook.dagger.scope.PerActivity;
+import br.com.mgalhardo.guidebook.presentation.ui.guidelist.GuideListActivity;
+import dagger.Subcomponent;
+
+@PerActivity
```

Comment:
Subcomponents should not be associated with the same scope as an ancestor. More info here: http://google.github.io/dagger/subcomponents.html